tests/modules/moduleparameter.c

```diff
@@ -0,0 +1,27 @@
+#include "valkeymodule.h"
+#include <stdio.h>
+#include <stdlib.h>
+#include <ctype.h>
+#include <string.h>
+
+int test_module_update_parameter(ValkeyModuleCtx *ctx,
+                                 ValkeyModuleString **argv, int argc) {
+  ValkeyModule_UpdateRuntimeArgs(ctx, argv, argc);
```

Comment:
Check that ValkeyModule_UpdateRuntimeArgs returns VALKEYMODULE_OK here.

Reply:
Update to:  return ValkeyModule_UpdateRuntimeArgs(ctx, argv, argc);

Reply:
Yes, or

```suggestion
  int res = ValkeyModule_UpdateRuntimeArgs(ctx, argv, argc);
  assert(res == VALKEYMODULE_OK);
```

Reply:
The command still needs to send a reply to the client, right?

Reply:
I just have one statement as return ValkeyModule_ReplyWithSimpleString(ctx, "OK");